CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+Successful governance actions are applied on an epoch boundary (they are **enacted**).
+
+One of the protocol parameters is sufficiently significant to merit special attention: changing the major protocol version enables Cardano to enact controlled hard forks. This type of update, therefore, has a special status amongst the possible protocol parameter updates.
+
+### Shortcomings of the Shelley Governance Design
+
+The current design was intended to provide a simple, transitional approach to governance.
+This proposal aims to address a number of shortcomings with that design.
+
+1. It gives no room for active on-chain participation of Ada holders. While changes to the protocol are usually the results of discussions with selected community actors, the process is driven mainly by the founding entities.
+Ensuring everyone can voice their concern is cumbersome and can be perceived as arbitrary at times.
+
+2. Movements from the treasury can be hard to track and constitute a critical and sensitive topic.
+It is important to have more transparency and more layers of control over these movements.
+
+3. While they need to be treated specially by SPOs, hard forks are not differentiated from other protocol parameter changes.
+
+4. Finally, while there's currently a somewhat common vision for _Cardano_ that is shared by its founding entities and many community members, there is no clearly defined document where these guiding principles are recorded. It makes sense to leverage the Cardano blockchain to record the ethos of the project itself in an immutable fashion, as the formal Cardano Constitution.
+
+## Specification
+
++ [The Cardano Constitution](#the-constitution)
++ [The Constitutional Committee](#the-constitutional-committee)
+  - [Initial Committee](#initial-committee)
+  - [Replacing The Committee](#replacing-the-committee)
++ [Governance Actions](#governance-actions)
+  - [Ratification](#ratification)
+    * [Requirements](#requirements)
+    * [Lifecycle](#lifecycle)
+  - [Enactment](#enactment)
+  - [Content](#content)
+  - [Governance Action IDs](#governance-action-ids)
++ [Votes](#votes)
+  - [Governance State](#governance-state)
+  - [Stale Votes](#stale-votes)
++ [Delegated Representatives (DReps)](#delegated-representatives--dreps-)
+  - [New Stake Distribution for DReps](#new-stake-distribution-for-dreps)
+  - [Definitions Surrounding Voting Stake](#definitions-surrounding-voting-stake)
+
+### The Cardano Constitution
+
+The Constitution is a text document that defines Cardano's shared values and guiding principles. At this stage, it is meant to be an informational document that unambiguously captures the Cardano core values. At a later stage, we can imagine the Constitution perhaps evolving into a smart-contract-based set of rules driving the entire governance framework. For now, however, the Constitution will remain an off-chain document whose hash digest value will be recorded on-chain.
+
+### The Constitutional Committee
```

Comment:
I didn't make a suggestion since I don't have one...

> We named it the Constitutional Committee since their stated purpose is to enforce the constitution, which in theory could involve any of the listed governance actions.

Okay, this does actually make sense indeed. I guess maybe this goes back to the question about hard vs soft statements about the committee: in terms of the _actual rules_ they're just "another body who has to assent to changes and who can be removed by a supermajority", but there is this social overtone that they're supposed to do it in a particular way :thinking: